RELEASE.md

```diff
@@ -1,11 +1,27 @@
-# How to release
+## Overview
```

Comment:
Great guide! [Rendered](https://github.com/dottxt-ai/outlines-core/blob/maturin/RELEASE.md)

Reply:
Good to know it's turned out to be clear